modules/bullet/SCsub

```diff
@@ -200,6 +200,11 @@ if env["builtin_bullet"]:
     thirdparty_sources = [thirdparty_dir + file for file in bullet2_src]
 
     env_bullet.Prepend(CPPPATH=[thirdparty_dir])
+    # Treat Bullet headers as system headers to avoid raising warnings. Not supported on MSVC. (TODO SWITCH: devkitA64 issue prevents this from working)
```

Comment:
This is mostly a reminder for me, to look at this later.

Reply:
Does this require a change?